_guides/software-request-guide.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Requesting Software at GSA
+category: Team
+---
+
+  <a href="http://codepen.io/saracope/full/aWPePG/" target="_blank"><img src="{{ site.baseurl }}/assets/img/guides/software-request-process-map.png"
+    alt="Software Request Process Map"
+    style="float: left;  margin-right: 10px;"></a>
+
+### Introduction
+
+Occasionally you may need to use software that is not already available to you. We've created this resource to guide you in the process of requesting software at GSA. Follow these simple steps below and view our interactive [process map](http://codepen.io/saracope/full/aWPePG/){:target="_blank"} to get started.
+
+### Research
+Once you have identified a need, discuss this with your supervisor and colleagues to get more information about what tool may be best to use. Be sure check if the software you want is available in the [GSA EA Analytics & Reporting](https://ea.gsa.gov/#!/itstandards) (GEAR) IT Standards list. Note: The GEAR site is only available within the GSA network. Once you have identified the software you would like, you will need to take some time to get all the information you will need to request it.
+<br>
+<br>
+Resources: [GEAR](https://ea.gsa.gov/#!/itstandards)
+
+### Gather
+You will need several pieces of information when completing a software request. Things like purpose, cost, and download link among others. Follow [this checklist](https://docs.google.com/a/gsa.gov/spreadsheets/d/1zIyZGdmEuWUeuko7yEaupETz2YGJQpOumHwiUntul64/edit?usp=sharing){:target="_blank"} to be sure you've gathered all the info you will need.
```

Comment:
I'm a big fan of checklists, though the information there is a bit redundant with some on this page. Might consider how to consolidate/deduplicate them.